collections/_article/dev-snapshot-godot-3-6-beta-2.md

```diff
@@ -0,0 +1,84 @@
+- Input: Augment the `InputEvent` class with a `CANCELED` state ([GH-76715](https://github.com/godotengine/godot/pull/76715)).
+- Physics: Fix RigidDynamicBody gaining momentum with bounce ([GH-76216](https://github.com/godotengine/godot/pull/76216)).
+- Porting: Add benchmark logic ([GH-71875](https://github.com/godotengine/godot/pull/71875)).
+- Porting: Android: Enable granular control of touchscreen related settings ([GH-73692](https://github.com/godotengine/godot/pull/73692)).
+- Porting: Android: Update the gradle build tasks to generate play store builds ([GH-74583](https://github.com/godotengine/godot/pull/74583)).
+- Porting: Android: Fix UI responsiveness to touch taps ([GH-75699](https://github.com/godotengine/godot/pull/75699)).
+- Porting: Android: Fix null in Android text entry system ([GH-75992](https://github.com/godotengine/godot/pull/75992)).
+- Porting: Android: Downgrade Android gradle plugin to version 7.2.1 ([GH-76329](https://github.com/godotengine/godot/pull/76329)).
+- Porting: Android: Allow concurrent buffering and dispatch of input events ([GH-76400](https://github.com/godotengine/godot/pull/76400)).
+- Porting: Android: Fix input ANR in the Godot Android editor ([GH-76981](https://github.com/godotengine/godot/pull/76981)).
+- Porting: Linux: Don't use udev for joypad hotloading when running in a sandbox ([GH-76961](https://github.com/godotengine/godot/pull/76961)).
+- Rendering: Consistent render ordering for CanvasLayers ([GH-69952](https://github.com/godotengine/godot/pull/69952)).
+- Rendering: Batching: Add MultiRect command ([GH-68960](https://github.com/godotengine/godot/pull/68960)).
+- Rendering: Fix Polygon2D skinned bounds (for culling) ([GH-75612](https://github.com/godotengine/godot/pull/75612)).
+- XR: Disable blending before blitting to framebuffer from WebXR ([GH-76072](https://github.com/godotengine/godot/pull/76072)).
+- Thirdparty: bullet 3.25.
+- Documentation updates.
+
+This release is built from commit [68c507f59](https://github.com/godotengine/godot/commit/68c507f59b05c4e53411585c9d4a3d1988717de9).
+
+## Downloads
+
+The downloads for this dev snapshot can be found directly on our repository:
+
+- [Standard build](https://downloads.tuxfamily.org/godotengine/3.6/beta2/) (GDScript, GDNative, VisualScript).
+- [Mono build](https://downloads.tuxfamily.org/godotengine/3.6/beta2/mono/) (C# support + all the above). You need to have dotnet CLI or MSBuild installed to use the Mono build. Relevant parts of Mono **6.12.0.182** are included in this build.
+
+## Bug reports
+
+As a tester, you are encouraged to [open bug reports](https://github.com/godotengine/godot/issues) if you experience issues with this release. Please check the [existing issues on GitHub](https://github.com/godotengine/godot/issues) first, using the search function with relevant keywords, to ensure that the bug you experience is not already known.
```

Comment:
We changed it to "we encourage you" as of dev 1:

```suggestion
As a tester, we encourage you to [open bug reports](https://github.com/godotengine/godot/issues) if you experience issues with this release. Please check the [existing issues on GitHub](https://github.com/godotengine/godot/issues) first, using the search function with relevant keywords, to ensure that the bug you experience is not already known.
```

Reply:
Fixed, as well as for 3.6 beta 1.